carbonmark-api/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@klimadao/carbonmark-api",
-  "version": "6.1.2",
+  "version": "6.2.1",
```

Comment:
please run `npm version minor` from the carbonmark-api dir (or run `npm install`)